go/worker/txnscheduler/algorithm/batching/batching.go

```diff
@@ -17,9 +17,6 @@ const (
 	// Name of the scheduling algorithm.
 	Name = "batching"
 
-	// CfgMaxBatchSize configures the max batch size.
-	CfgMaxBatchSize = "worker.txnscheduler.batching.max_batch_size"
-
 	cfgMaxQueueSize      = "worker.txnscheduler.batching.max_queue_size"
```

Comment:
`cfgMaxQueueSize` &  `cfgMaxBatchSizeBytes` should probably also be consistent (sorry missed them last time since we don't every actually modify these two at the moment)

Reply:
Probably not `cfgMaxQueueSize` but yeah `cfgMaxBatchSizeBytes` should be.